telecine/src/main/java/com/jakewharton/telecine/TelecineApplication.java

```diff
@@ -31,10 +32,28 @@
       Timber.plant(tree);
     }
 
-    objectGraph = ObjectGraph.create(new TelecineModule(this));
+    telecineModule = new TelecineModule(this);
+
+    initTelecineComponent();
+
+    getTelecineComponent().inject(this);
+  }
+
+  void initTelecineComponent() {
```

Comment:
I created the getters and setters for future roboletric unit tests so you
can override them. I can inline all of them if you want.
On Jun 14, 2016 7:12 AM, "Jake Wharton" <notifications@github.com> wrote:

> In
> telecine/src/main/java/com/jakewharton/telecine/TelecineApplication.java
> <https://github.com/JakeWharton/Telecine/pull/115#discussion_r66977533>:
>
> > @@ -31,10 +32,28 @@
> >        Timber.plant(tree);
> >      }
> >
> > -    objectGraph = ObjectGraph.create(new TelecineModule(this));
> > +    telecineModule = new TelecineModule(this);
> > +
> > +    initTelecineComponent();
> > +
> > +    getTelecineComponent().inject(this);
> > +  }
> > +
> > +  void initTelecineComponent() {
>
> inline this method. there's only one call site.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/JakeWharton/Telecine/pull/115/files/688b691721cbcb761fef90dbb8316eea24f1f9b3#r66977533>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABqMSEcIXTo1rXtRb-7orMjyz48Tp3gNks5qLrbfgaJpZM4I0-8d>
> .
>